tests/functional/lib/helpers.js

```diff
@@ -859,6 +859,56 @@ define([
       .then(testElementExists('.attached' + attachedId));
   });
 
+  /**
+   * Store the data sent for a WebChannel event into sessionStorage.
+   *
+   * @param {string} expectedCommand command to store data for.
+   * @returns {promise}
+   */
+  const storeWebChannelMessageData = thenify(function (expectedCommand) {
+    return this.parent
+      .executeAsync(function (expectedCommand, callback) {
+        function listener(e) {
+          var command = e.detail.message.command;
+          if (command === expectedCommand) {
+            const storedEvents = JSON.parse(sessionStorage.getItem('webChannelEventData')) || {};
+            storedEvents[command] = e.detail.message;
```

Comment:
I'm probably over-thinking this but do web channel messages ever occur more than once? Any mileage in storing an array of messages per command instead of clobbering?

Even as I type those, I'm saying to myself YAGNI. Feel free to ignore.